x/gov/client/utils/query.go

```diff
@@ -68,7 +69,9 @@ func QueryVotesByTxQuery(
 		fmt.Sprintf("%s='%s'", tags.ProposalID, []byte(fmt.Sprintf("%d", params.ProposalID))),
 	}
 
-	infos, err := tx.SearchTxs(cliCtx, cdc, tags)
+	// NOTE: SearchTxs is used to facilitate the txs query which does not currently
+	// support configurable pagination.
+	infos, err := tx.SearchTxs(cliCtx, cdc, tags, 1, 30)
```

Comment:
Apologies, I've approved the changes by mistake. I actually intend to drop few more comments.
Can we bump up the default items number a bit? I think a sane default should be between 100 and 200.
And it'd be great to move such value into a const.

Reply:
I think the reasoning here was that Tendermint tx query returns by default a limit of 30. But maybe you're right we should have at least 50 or 100. Thoughts @ValarDragon @cwgoes ?

> And it'd be great to move such value into a const.

Agree

Reply:
needs change tendermint as well, how about let me do it in a separate PR (but same issue)?

https://github.com/tendermint/tendermint/issues/3065

https://github.com/tendermint/tendermint/blob/1e1ca15bccedc16c2452c73745c3c2b5df91ae96/rpc/core/pipe.go#L151-L156

Reply:
Seems fair

Reply:
As I said, I'm ok with addressing the rest of work in a separate PR, though please don't hardcode values, use const instead

Reply:
Looks like these still need to be constants?